examples/wms-time.js

```diff
@@ -0,0 +1,64 @@
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.MapQuest');
+goog.require('ol.source.TileWMS');
+
+var startDate = new Date(Date.parse('2012-01-01T19:00:00Z'));
+var frameRate = 0.5; // frames per second
+var animationId = null;
+
+var counter = 0;
```

Comment:
Defined but never used.  ESLint [caught this](https://travis-ci.org/openlayers/ol3/builds/103601244) after the merge.